src/App/Tracker/Controller/Hooks/ReceiveCommentsHook.php

```diff
@@ -143,11 +154,16 @@ protected function insertComment()
 		}
 		catch (\Exception $e)
 		{
-			$this->logger->error(
-				'Error loading the database for comment '
-				. $this->hookData->issue->number
-				. ':' . $e->getMessage()
+			$logMessage = sprintf(
+				'Error loading the database for comment %d:%s',
+				$this->hookData->issue->number,
+				$e->getMessage()
 			);
+			$this->logger->error($logMessage);
+			$this->setStatusCode($e->getCode());
+			$this->response->error = $logMessage;
+
+			return false;
```

Comment:
The try/catch was more for the table load than the event.  We don't need to return a false here because the only thing that fails is the plugin event, not a "critical" failure as with other conditions.